features.txt

```diff
@@ -252,6 +252,10 @@ resizable-arraybuffer
 # https://github.com/tc39/proposal-arraybuffer-transfer
 arraybuffer-transfer
 
+# ArrayBuffer detached
+# https://tc39.es/proposal-arraybuffer-transfer/
+arraybuffer-detached
```

Comment:
Could we re-use the flag `arraybuffer-transfer` for these tests? `arraybuffer-detached` may be incorrectly assumed to mean "tests involving detached ArrayBuffers." `arraybuffer-prototype-detached` would be more clear in that sense, but as a general rule, I prefer to use a single feature flag for a given proposal. It's an easier policy to apply regardless of proposal size, and more granularity doesn't buy much in most cases: folks that want to filter for certain APIs can lean on the directory structure.

Reply:
We could, but they're three separate methods, and there's no guarantee they'll be implemented independently.

Reply:
Please reuse the same `arraybuffer-transfer` flag. There is a 99.9% chance they will be implemented together, since implementations implement proposals at once, not cherrypick individual methods out of proposals...

Reply:
I don't agree that's the main point of the flags _now_, but either way I've updated to reuse the transfer flag.